app/services/npq/build_application.rb

```diff
@@ -70,6 +70,7 @@ def teacher_catchment_iso_country_code
         country.alpha3
       else
         Sentry.capture_message("Could not find the ISO3166 alpha3 code for #{teacher_catchment_country}.")
```

Comment:
Could we also silence those in sentry for now, I reckon we don't need them adding more noise for the time being, until we get to tackle them?